File: Competitions/UC/Berkeley-2024/contest/contest/caliconstruction/templates/caliconstruction_template.java
import java.io.*;

class Solution {
    /**
     * Return the minimum number of sets of CALICO blocks needed to
     * form S, or -1 if it is impossible.
     *
     * S: a string
     */
    static int solve(String S) {
        return 1;
    }

    static BufferedReader in = new BufferedReader(new InputStreamReader(System.in));
    static PrintWriter out = new PrintWriter(System.out);

    public static void main(String[] args) throws IOException {
        int T = Integer.parseInt(in.readLine());
        for (int i = 0; i < T; i++) {
            String S = in.readLine();
            out.println(solve(S));
        }
        out.flush();
    }
}

File: Competitions/UC/Berkeley-2024/contest/contest/caliconstruction/templates/caliconstruction_template.py
def solve(S: str) -> int:
    """
    Return the minimum number of sets of CALICO blocks needed to
    form S, or -1 if it is impossible.

    S: a string
    """
    return 1


def main():
    T = int(input())
    for _ in range(T):
        S = input()
        print(solve(S))


if __name__ == '__main__':
    main()
